src/data/emission/ailleurs-312/manifest.json

```
@@ -0,0 +1,9 @@
+{
+  "title": "Tricholome savonneux",
+  "slug": "Tricholome savonneux",
+  "authors": "Julie aka Paxille",
+  "releasedAt": "2022-01-02",
+  "type": "Ailleurs",
+  "isPublic": true
```

```suggestion
  "isPublic": false
```

Comme ça quand on merge la PR c'est déployé en invisible sur le site, et @johan-ci peut décider de la mise à ligne quand il veut.